content/practices/team-shared-history.md

```diff
@@ -0,0 +1,49 @@
+---
+date: 2018-06-28T00:00:00-05:00
+title: Team Shared History
+draft: false
+stages: ['delivery']
+---
+
+## What is it?
+
+A method of helping a team start to become cohesive, develop a shared sense of history, and a shared vision of the future. This simple exercise starts with a facilitator organizing the participants in order of their hire date with the company. Then a timeline is drawn and each team member marks their hire date on the timeline. Next, team members note on the timeline interesting things which may have occurred near their time of hire. Finally, the team member who has the most seniority will "tell the story" of the team's history and perhaps embellish with additional context around what has been going on around the organization.
```

Comment:
- don't use "method". Use "Practice". 
- Don't describe the "How" facilitation in this section. Concentrate on the outcomes desired and outputs created.

Reply:
@InfoSec812 I believe that in the spirit of positive intent and correct content. let's merge this and add in additional changes requested as more issues and update it appropriately.